Blog: How to craft product survey questions (#6520)
PostHog/posthog.com: contents/blog/product-survey-questions.md

---
title: 18 product survey questions to help build a great product
date: 2023-08-17
author: ["ian-vanagas"]
showTitle: true
rootpage: /blog
sidebar: Blog
hideAnchor: true
featuredImage: ../images/blog/posthog-blog-image.png
featuredImageType: full
category: Product growth
tags:
 - Guides
 - Explainers
---

Product surveys help you understand what users are thinking and feeling. Getting to these thoughts and feeling require you to ask great questions.

To help you do this, this post goes over the importance of goals for product survey questions, the principles of great questions, question examples, the question real companies ask, and how to use responses. 

> > **Want to run surveys?** See our tutorial on [how to create custom surveys](/tutorials/survey) with options like open text, multiple choice, or custom components.

Reviewer: ```suggestion
> **Want to run surveys?** See our tutorial on [how to create custom surveys](/tutorials/survey) with options such as open text, multiple choice, or custom components.
```

## Start with a goal for your product survey

Like an [A/B test](/blog/ab-testing-guide-for-engineers), you should start your product survey process with a goal in mind. A product survey, by definition, has a goal related to the product. This could include improving a feature, prioritizing feature futures, or optimizing critical flows.

> Because surveys provide open, qualitative feedback from users, goals don’t need to be specific. For example, they don’t need to be a single metric you aim to improve. The goal of an individual survey should align with broader goals for a product and ultimately work towards improving it.

Reviewer: ```suggestion
Goals don’t need to be specific. For example, they don’t need to be a single metric you aim to improve. The goal of an individual survey should align with broader goals for a product and ultimately work towards improving it.
```

> Having a goal enables you to decide how the survey runs and who it targets. For example, if the goal is to improve the onboarding flow, it should run soon after users complete the flow. It shouldn’t run for long-time users who completely forget about the onboarding process or went through an entirely different process.

Reviewer: ```suggestion
A goal enables you to decide how the survey runs and who it targets. For example, if the goal is to improve the onboarding flow, it should run soon after users complete the flow. It shouldn’t run for long-time users who completely forget about the onboarding process or went through an entirely different process.
```

## Product survey question examples

With those principles in mind, here are examples of product survey questions as inspiration. We broke them down into four categories based on the type of goals they help achieve.

![Surveys](../images/blog/product-survey-questions/surveys.png)

### 1. Usage

**Goal:** Understand how users use the product in ways that aren’t clear from the data.

- How do you use X feature?
- What is your use case for X feature?
- What problem are you trying to solve?
- Why did you choose our product?

### 2. Opinion

**Goal:** Understand what users are thinking and feeling about your product or feature.

- What’s frustrating about X feature?
- What do you like/dislike about X feature?
- How would you feel if you could no longer use the product?
- [How likely is it that you would recommend the product to a friend or colleague?](/tutorials/nps-survey)
- How easy/difficult is the product to use?

### 3. Roadmap

**Goal:** Understand what to prioritize in your feature roadmap.

- What is missing from our product?
- How can we improve feature X?

> - What prevents you from achieving your goal with our product?

Reviewer 2: Is this leading, maybe? I dunno.

- How useful would X feature be to you?
- What new features do you want to see the most?

> ### 4. Competitors

Reviewer 2: Comparative feels easier to parse for me

- [Zola](https://review.firstround.com/heres-why-zola-cherishes-its-nps-detractors-as-much-as-its-promoters) sent NPS surveys every month since its launch. They focused on feedback from detractors over promoters to guide product changes.

This shows the diversity of questions, companies, and use cases for product surveys. Each of these companies utilized surveys to prioritize future development, monitor user satisfaction, and ultimately, make data-informed decisions to improve their product.

## Using surveys to find product-market fit

The real companies listed above are big tech companies with product-market fit, but what about companies still searching for it? Product surveys are a critical part of the process.

In the search for product-market fit, surveys are often more helpful than stats:

- Stats show you what is happening, but surveys help you understand why it is happening. 

> - Stats are often lagging indicators of product-market fit, they begin improve after you have it. Surveys are leading indicators, feedback will hint that you have it.

Reviewer: ```suggestion
- Stats are often lagging indicators of product-market fit, they improve after you have it. Surveys are leading indicators, feedback will hint that you have it.
```

- Surveys help you identify and prioritize the next steps to improving your product. They are more actionable than stats at the early stage.

Using the middle three levels of [the product-market fit game](/blog/product-market-fit-game), we can identify example questions you would use to advance to the next level:

**Validate the problem by talking to users**

- What is the problem you’re trying to solve?
- What do you currently do to solve this problem?
- What are your frustrations with the current solutions?

**Get users to use your product**

- What's your use case for our product?
- What's missing from our product?
- What prevents you from achieving your goal with our product?

**Keep users coming back**

- What is painful/frustrating about our product?
- Why did you choose our product?
- How can we improve feature X?
- [How would you feel if you could no longer use the product?](/blog/measure-product-market-fit#indicator-2-pmf-survey)

## Utilizing your product survey results

> Once you [implement your product survey](/tutorials/survey) and get all the data from it, you need to put it to work. Asking questions and collecting responses is useless if you don’t do anything with them.

Reviewer 2: ```suggestion
Once you [implement your product survey](/tutorials/survey) and get all the data from it, you need to put it to work. You can: 
```

> First, you can use product survey data as a jumping-off point for more research. In PostHog, you can easily connect respondents to their usage data or look into usage trends based on responses. You can also use the responses to identify [users for interviews](/tutorials/feedback-interviews-site-apps) and dig into details further.

Reviewer 2: ```suggestion
1. **Do more research** In PostHog, it's easy to connect respondents to their usage data, or look into usage trends based on responses. You can also use the responses to identify [users for interviews](/tutorials/feedback-interviews-site-apps) and dig into details further.
```

> Second, you can monitor satisfaction with your product and features. Running repeated surveys asking the same questions can give you an idea of user perceptions over time. You can use this to track the impact of changes and identify areas requiring change.

Reviewer 2: ```suggestion
2. **Start monitoring customer satisfaction** – Run repeated surveys asking the same questions to give you an idea of user perceptions over time. You can use this to track the impact of changes and identify areas requiring change.
```

> Third and finally, you can prioritize future development and make data-driven decisions. Ultimately, the goal of product tools (and PostHog) is to help you build better products. Knowing what users think about your product and what they want to see next is a great way to do that.

Reviewer 2: ```suggestion
3. **Update your product roadmap** – Finally, you can prioritize future development and make data-driven decisions. Ultimately, the goal of product tools (and PostHog) is to help you build better products. Knowing what users think about your product and what they want to see next is a great way to do that.
```

> 

Reviewer 2: ```suggestion

As an all-in-one platform, PostHog is uniquely placed to do all the above. In addition to surveys, we also do product analytics, session replay, feature flags, and A/B testing. If this sounds useful to you, [check us out](https://posthog.com/).
```

---

> title: 18 product survey questions to help build a great product

Reviewer 2: ```suggestion
title: "How to write great product survey questions (with examples)"
```

Thoughts on his headline, @ivanagas?

Author: Good with me
